devicemodel/hw/pci/xhci.c

```diff
@@ -2024,7 +1984,6 @@ pci_xhci_cmd_address_device(struct pci_xhci_vdev *xdev,
 		dev_ctx->ctx_slot.dwSctx2, dev_ctx->ctx_slot.dwSctx3);
 
 	dev = XHCI_SLOTDEV_PTR(xdev, slot);
-	assert(dev != NULL);
```

Comment:
Will it be NULL?

Reply:
There are codes to process the NULL case before this assert.

1943         dev = xdev->slots[slot];
1944         if (!dev) {